poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -512,16 +516,19 @@ public void checkAndUpdateIntegrationStatus(FacilitySoftwareConfig powerIQ, Stri
          }
          //the pduId and sensorId are form the powerIQ system.
          if (sensor.getPduId() != null) {
-            justificationfieldsForSensor.put(Pdu_ID, sensor.getPduId().toString());
+            justificationfieldsForSensor.put(FlowgateConstant.PDU_ID_FROM_POWERIQ, sensor.getPduId().toString());
          }
-         justificationfieldsForSensor.put(Sensor_ID, sensor.getId() + "");
+         justificationfieldsForSensor.put(FlowgateConstant.SENSOR_ID_FROM_POWERIQ, sensor.getId() + "");
          asset.setAssetName(sensor.getName());
          asset.setJustificationfields(justificationfieldsForSensor);
          asset.setSerialnumber(sensor.getSerialNumber());
          asset.setAssetSource(assetSource);
          asset.setCategory(AssetCategory.Sensors);
          asset.setSubCategory(subCategoryMap.get(sensor.getType()));
          asset.setCreated(System.currentTimeMillis());
+         if(sensor.getPosition() != null) {
+            asset.setMountingSide(sensorMountingSide.get(sensor.getPosition()));
```

Comment:
I would suggest that you should use case insensitive string to identify the outlet or inlet.
like sensor.getPosition.toUppercase  or tolowercase. in that way it will be more robust.